Source/Core/CircleGeometry.js

```diff
@@ -156,5 +158,24 @@ define([
         return EllipseGeometry.createGeometry(circleGeometry._ellipseGeometry);
     };
 
+    CircleGeometry._createShadowVolume = function(circleGeometry, minHeightFunc, maxHeightFunc) {
```

Comment:
Throughout for `createShadowVolume`, it's probably better to drop the underscore and use `@private`, right?